modules/run-vault/run-vault

```diff
@@ -392,9 +401,12 @@ KillSignal=SIGINT
 Restart=on-failure
 RestartSec=5
 TimeoutStopSec=30
+StartLimitInterval=60
 StartLimitIntervalSec=60
 StartLimitBurst=3
 LimitNOFILE=65536
+LimitMEMLOCK=infinity
```

Comment:
I'm curious what are these settings tweaks doing?

Reply:
This is mentioned [here](https://learn.hashicorp.com/tutorials/vault/deployment-guide#step-3-configure-systemd): `LimitNOFILE`, `LimitMEMLOCK` - Limit the number of file descriptors to the configured value and prevent memory being swapped to disk

Reply:
Ah, good to know, thanks!